modules/accounting/assets/src/admin/components/transactions/sales/InvoiceSingleContent.vue

```diff
@@ -58,7 +58,7 @@
             </div>
 
             <div class="wperp-invoice-table">
-                <table class="wperp-table wperp-form-table invoice-table">
+                <table class="wperp-single-sales-table wperp-form-table invoice-table">
```

Comment:
Again no need to introduce a new table class as long as you can modify the existing class according to your need.